misc/python/materialize/feature_benchmark/scenarios/benchmark_main.py

```diff
@@ -1949,10 +1953,11 @@ def benchmark(self) -> BenchmarkingSequence:
 class StartupTpch(Scenario):
     """Measure the time it takes to restart a Mz instance populated with TPC-H and have all the dataflows be ready to return something"""
 
-    SCALE = 1.2  # 25ish objects of each kind
-    FIXED_SCALE = (
-        True  # Can not scale to 100s of objects, so --size=+N will have no effect
-    )
+    # Runs ~3 hours with SCALE = 1.2
+    SCALE = 0.1  # 1 object of each kind
```

Comment:
This comment seems to indicate that the `SCALE` should be 1.2?

Reply:
It's supposed to explain why we are not using SCALE = 1.2 anymore, 3 hours is too long.